core/audits/seo/is-crawlable.js

```diff
@@ -24,7 +24,7 @@ const UIStrings = {
   failureTitle: 'Page is blocked from indexing',
   /** Description of a Lighthouse audit that tells the user *why* allowing search-engine crawling of their page is beneficial. This is displayed after a user expands the section to see more. No character length limits. The last sentence starting with 'Learn' becomes link text to additional documentation. */
   description: 'Search engines are unable to include your pages in search results ' +
-      'if they don\'t have permission to crawl them. [Learn more about crawler directives](https://web.dev/is-crawable/).',
```

Comment:
oof on the mispell

Reply:
Ha, well spotted. I was confused at first how it worked at all (turns out https://github.com/GoogleChrome/web.dev/pull/5407 added a redirect) and then worried for a second that we had our audit ID wrong all this time.